contracts/hardhat.config.js

```diff
@@ -195,6 +205,8 @@ module.exports = {
         process.env.FORK === "true"
           ? isHoleskyFork
             ? HOLESKY_DEPLOYER
+            : isBaseFork
+            ? BASE_DEPLOYER
```

Comment:
I don't agree with this change personally. If we change the governor on fork, we are deploying/testing the contracts in a state that's gonna be completely different from mainnet.

Reply:
Agreed, this has been removed.